decorators/headerFooter.js

```diff
@@ -14,15 +14,15 @@ function headerFooter(ComposedComponent) {
       if (!templates || !templates[type]) {
         return null;
       }
-      let className = cx(this.props.cssClasses[type], `ais-${type}`);
+      let cssClass = cx(this.props.cssClasses[type], `ais-${type}`);
```

Comment:
We're using `className={cssClasses.root}`, made more sense to me to have `className={cssClass}` to match these.